api/src/main/java/org/pmiops/workbench/BlockscoreService.java

```diff
@@ -1,11 +1,18 @@
 package org.pmiops.workbench;
 
+import com.blockscore.models.Address;
 import com.blockscore.models.PaginatedResult;
 import com.blockscore.models.Person;
+import org.pmiops.workbench.model.BlockscoreVerificationStatus;
 
 /**
  * Encapsulate API for interfacing with Blockscore identity verification service.
  */
 public interface BlockscoreService {
   public PaginatedResult<Person> listPeople();
```

Comment:
What is this method for? When do we envision using it?

Reply:
It's only used in the integration test right now. I would use a `createPerson` call, but as far as I can tell, there is no way to delete them, so the verification requests would just accumulate without bound if used in integration.